runsc/boot/config.go

```diff
@@ -112,6 +113,18 @@ func MakeWatchdogAction(s string) (watchdog.Action, error) {
 	}
 }
 
+// MakeRefsLeakMode converts type from string
+func MakeRefsLeakMode(s string) (refs.LeakMode, error) {
+	switch strings.ToLower(s) {
+	case "disabled":
+		return refs.NoLeakChecking, nil
+	case "warning":
+		return refs.LeaksLogWarning, nil
+	default:
```

Comment:
No way to configure traces?

Reply:
Hi @iangudger, there is as you can see in my initial commit here https://github.com/google/gvisor/pull/655/commits/607be0585fdc659ec3c043c989a8a6f86fcc14db#diff-a21a7e36b6725efe451f1883bc1a38eeR123. I removed it following review comments provided. Can add it back if it is needed

Reply:
My bad, I misunderstood what trace mode did. It's actually the most important one :-)